### ISSUE TITLE
Add statement hint to disable DDL statement in emulator.

### ISSUE THREAD

**Issue by hughack** [GoogleCloudPlatform/cloud-spanner-emulator#92]
Some statements are not supported by the emulator, e.g.:

- `CREATE CHANGE STREAM...`
- Columns with `DEFAULT` values.

It would be good to be able to add a statement hint that will skip a statement in the emulator.

For example a SQL script could be like this:

```SQL
CREATE TABLE MyTable ...;

@{spanner_emulator.skip_statement=true}
CREATE CHANGE STREAM ...;
```

**Comment by gauravsnj** [comment]
Its available in version 1.4.8

**Comment by hughack** [comment]
Since this is closed I've created a new feature request to better articulate this feature request #93 .

Change streams and DEFAULT are examples, but it is a generic issue with any new feature that gets added and isn't supported. There are legitimate other use cases unrelated to feature parity.
